Sources/Utils.php

```diff
@@ -19,15 +19,14 @@
 class Utils
 {
 	use BackwardCompatibility;
-
```

Comment:
Please restore the missing blank line.

Can also remove the 'func_names' subarray from the $backcompat property below.

Reply:
I only targeted one function/method pair from this file for testing very early on. The single function has been defined as first in the list will move when I get to that class.